docs/docs-content/clusters/edge/local-ui/host-management/edit-user-data.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_label: "Edit User Data"
+title: "Edit User Data"
+description: "Instructions for editing user data in Local UI."
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["edge"]
+---
+
+The **user-data** file is a YAML file that contains installer configuration for the Palette agent. This file is usually
+prepared before the EdgeForge process and is written into the installer ISO in appliance mode deployments. In agent mode
+deployments, the user data is provided directly to the agent mode installation script. For more information about
+EdgeForge, refer to [EdgeForge Workflow](../../edgeforge-workflow/edgeforge-workflow.md). For more information about
+deployment modes, refer to [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+Local UI allows you to make edits to most fields in the **user-data** file after installation has completed before
+cluster creation. All fields exposed in the form in Local UI have corresponding parameters in the installer reference.
+For more information about the fields available in the installer user data, refer to
+[Installer Reference](../../edge-configuration/installer-reference.md).
+
+After an edit has been made, the new settings will apply after the host reboots.
+
+:::preview
+
+:::
+
+## Limitations
+
+- The following fields cannot be edited in Local UI. If you need to use these settings, you must set them correctly
+  during EdgeForge or provide them to the installation script if you are using agent mode.
+  - `stylus.site.deviceUIDPaths`
+  - `stylus.site.tagsFromFile`
+  - `stylus.site.tagsFromScript`
+  - `stylus.localUI.port`
+  - `stylus.includeTui`
+  - `stylus.debug`
+  - `stylus.trace`
+  - `stylus.disablePasswordUpdate`
+  - `stylus.imageRedirectWebhook`
+  - `install.bind_mounts`
+
+## Prerequisites
+
+- You have set `stylus.featureGate` to `UserDataForm` in your installer user data during EdgeForge or provide the user
+  data to the installation script if you are using agent mode. For example, the following configuration enables editing
+  user data in Local UI.
```

Comment:
@lennessyy are you trying to say that users need to set `stylus.featureGate` to `UserDataForm` ? The paragraph confused me a bit.

Maybe something like.

Set the user-data parameter `stylus.featureGate` to the value `UserDataForm`. This applies to both Applicance Mode, and Agent Mode.

Reply:
The other thing I noticed is that we don't have the parameter defined in Edge Installer Configuration Reference page.

Reply:
I wanted to emphasize **when** they set it, though, because this whole feature is about editing user data in the UI, and this is not like they can change this in the UI later.